sass/_theme_font_local.sass

```diff
@@ -38,4 +38,5 @@
   font-family: 'Roboto Slab'
   font-style: normal
   font-weight: 700
-  src: local('Roboto Slab Bold'), local('RobotoSlab-Bold'), url(../fonts/RobotoSlab-Bold.ttf) format('truetype')
+  src: url('../fonts/RobotoSlab/roboto-slab-v7-bold.eot')
+  src: url('../fonts/RobotoSlab/roboto-slab-v7-bold.eot?#iefix') format('embedded-opentype'), url('../fonts/RobotoSlab/roboto-slab-v7-bold.woff2') format('woff2'), url('../fonts/RobotoSlab/roboto-slab-v7-bold.woff') format('woff'), url('../fonts/RobotoSlab/roboto-slab-v7-bold.ttf') format('truetype')
```

Comment:
While testing this release, I found that this change from #642 was reverted probably in a faulty merge conflict https://github.com/rtfd/sphinx_rtd_theme/pull/639#pullrequestreview-126491151